src/python/turicreate/toolkits/object_detector/_tf_image_augmenter.py

```diff
@@ -151,7 +151,6 @@ def resize_turicreate_image(image, output_shape):
         raise Exception("Non-supported resize method.")
 
     image_clipped = tf.clip_by_value(image_scaled, 0.0, 1.0)
-    annotation = tf.clip_by_value(annotation, 0.0, 1.0)
```

Comment:
The coordinates change if not in normalized format? Is it already in normalized format?

Reply:
I believe the bounding-box coordinates are normalized to [0,1] at this point, but the class labels are just indices from 0 .. n-1

Reply:
I'm not sure. I don't have a deep understanding of this code. I believe at this point the coordinates should all be relative and between zero and one, and the confidences should all be ones.

Certainly, clipping is the wrong behavior for the last dimension of `annotation` (i.e. the dimension which hold the label id). Clipping *might* make sense for the other dimensions, since they should all be between zero and one. However it just seems wrong to me. If we're given bad data, things should break. It seems like normalizing/clipping could hide problem; I think that's exactly what was happening when we had more than two classes.

What do you think?

Reply:
@nickjong - I didn't see your message before sending my message. I believe you are correct. 

@shreyajain17 - I don't think clipping/normalizing makes sense here. What do you think?

Reply:
To be clear, I only know that the coordinates are normalized, not in image coordinates, when passed from C++. That doesn't guarantee that the are literally in [0,1], or that any of the augmentation here can cause these coordinates to fall outside this range. But even then, I believe the actual network is largely robust to that anyway??

Reply:
My only concern was that if the image is resized, the coordinates of the bounding box must be adjusted correctly or the network gets wrong labels. The coordinates falling outside 0 to 1 is fine as long as the coordinates after rising are correct!